pkg/logcli/query/query.go

```diff
@@ -120,20 +117,30 @@ func (q *Query) printStream(streams logql.Streams, out output.LogOutput) {
 		}
 	}
 
-	d := q.resultsDirection()
-	i := iter.NewStreamsIterator(streams, d)
+	// sort and display entries
+	allEntries := make([]streamEntryPair, 0)
```

Comment:
Good move, this wasn't really required as streams were already deduplicated by the querier.